indico/modules/events/settings.py

```diff
@@ -250,7 +251,8 @@ def get_themes_for(self, event_type):
     'start_dt_override': None,
     'end_dt_override': None,
     'organizer_info': '',
-    'additional_info': ''
+    'additional_info': '',
+    'contrib_duration': TimedeltaConverter.from_python(timedelta(minutes=20))
```

Comment:
instead of using `event_core_settings` i would add a new `contribution_settings` EventSettingsProxy in `indico/modules/contributions/__init__.py`.

also, set the converter in the `converters` arg instead of calling it manually